pytest_mergify/flaky_detection.py

```diff
@@ -165,7 +165,7 @@ def _fetch_context(self) -> _FlakyDetectionContext:
 
         return result
 
-    def detect_from_report(self, report: _pytest.reports.TestReport) -> bool:
+    def try_fill_metrics_from_report(self, report: _pytest.reports.TestReport) -> bool:
```

Comment:
This is just a renaming because "detecting" doesn't mean anything in this context